app/models/item.rb

```diff
@@ -45,7 +45,7 @@ def next_hold
     electric_battery: "electric (battery)"
   }
 
-  audited
+  audited except: :plain_text_description
```

Comment:
No need to track changes to this column: it's only used to make it easier to search for items.